l10n_eu_product_adr_local/views/adr_goods_packaging_group_views.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="adr_goods_packaging_group_view_tree" model="ir.ui.view">
```

Comment:
Please add a basic `kanban` view. And, then, a `form` view as well, which will become accessible through `kanban` view.